packages/shared/src/enums.ts

```diff
@@ -7,6 +7,10 @@ const arrayToMap = <const T extends readonly string[]>(
 ): ArrayToMap<T> =>
   Object.fromEntries(array.map((key) => [key, key])) as ArrayToMap<T>;
 
+export const InternalAssets = arrayToMap(['Flip', 'Usdc', 'Dot', 'Eth', 'Btc']);
```

Comment:
i decided to call this `InternalAssets` after all because using `ChainflipAsset` felt too generic when working with it (we would have a `Asset` and a `ChainflipAsset` in the same context). i think `InternalAsset` is a bit more descriptive and also easier to change if we need to (if you depend on something that is prefixed with `Internal` you better expect changes lol)